patterns/tornado.php

```diff
@@ -743,96 +958,147 @@ class="wp-block-cover__image-background wp-image-3742" alt=""
+                        alt=""
+                        class="wp-image-2371"
+                        title=""
+                /></figure>
                 <!-- /wp:image -->
             </div>
             <!-- /wp:group -->
             <!-- /wp:areoi/column -->
 
             <!-- wp:areoi/column {"block_id":"efedf348-a0ff-4ff0-b8d1-78c2bb2bf89a","lock":{"move":false,"remove":false},"className":"px-0"} -->
             <!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|20","right":"var:preset|spacing|20","bottom":"var:preset|spacing|20","left":"var:preset|spacing|20"}}},"layout":{"type":"constrained","justifyContent":"left"}} -->
-            <div class="wp-block-group"
-                style="padding-top:var(--wp--preset--spacing--20);padding-right:var(--wp--preset--spacing--20);padding-bottom:var(--wp--preset--spacing--20);padding-left:var(--wp--preset--spacing--20)">
+            <div
+                class="wp-block-group"
+                style="padding-top: var(--wp--preset--spacing--20); padding-right: var(--wp--preset--spacing--20); padding-bottom: var(--wp--preset--spacing--20); padding-left: var(--wp--preset--spacing--20)"
+            >
                 <!-- wp:heading {"level":3,"fontSize":"small"} -->
                 <h3 class="wp-block-heading has-small-font-size"><strong>Preparing for a tornado</strong></h3>
                 <!-- /wp:heading -->
 
                 <!-- wp:group {"style":{"spacing":{"padding":{"bottom":"var:preset|spacing|20"}}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
-                <div class="wp-block-group" style="padding-bottom:var(--wp--preset--spacing--20)">
+                <div
+                    class="wp-block-group"
+                    style="padding-bottom: var(--wp--preset--spacing--20)"
+                >
                     <!-- wp:areoi/icon {"block_id":"a410e022-07a3-4fe0-97c9-1bc83807f3fe","style":"text-dark","icon":"bi-link"} -->
-                    <div class="wp-block-areoi-icon areoi-icon"><i class="text-dark bi-link "
-                            style="font-size:24px"></i></div>
+                    <div class="wp-block-areoi-icon areoi-icon"
+                        ><i
+                            class="text-dark bi-link"
+                            style="font-size: 24px"
+                        ></i
+                    ></div>
                     <!-- /wp:areoi/icon -->
 
                     <!-- wp:paragraph -->
-                    <p><a
-                            href="https://www.canada.ca/en/services/policing/emergencies/preparedness/get-prepared/hazards-emergencies/tornados/how-prepare.html#bfr">Government
-                            of Canada - Preparing for a tornado</a></p>
+                    <p><a href="https://www.canada.ca/en/services/policing/emergencies/preparedness/get-prepared/hazards-emergencies/tornados/how-prepare.html#bfr">Government of Canada - Preparing for a tornado</a></p>
```

Comment:
This is what caused the spacing issue as it truncates the line and puts it on a new line.
Before: 
<img width="879" height="191" alt="image" src="https://github.com/user-attachments/assets/ad9f39fe-cd1f-4240-9a10-a83c491fdfe5" />
After:
<img width="866" height="148" alt="image" src="https://github.com/user-attachments/assets/afec901f-c9f5-421b-bdd0-03dd60c6259b" />
This did spacing issue didn't occur on the frontend so the screenshots have not changed despite being updated.